lib/response.rb

```diff
@@ -1,24 +1,38 @@
 class Response
-  attr_reader :verb, :path, :protocol, :host, :port, :origin, :accept
+  attr_accessor :status_code
+  attr_reader :verb, :path, :protocol, :host, :port, :origin,
+              :accept, :guess
 
   def initialize
-    @verb     = ""
-    @path     = ""
-    @protocol = ""
-    @host     = ""
-    @port     = ""
-    @origin   = ""
-    @accept   = ""
+    @status_code = "200 ok"
+    @guess       = nil
   end
 
   def process_lines(request_lines)
     @verb     = request_lines[1][0]
     @path     = request_lines[1][1]
     @protocol = request_lines[1][2]
-    @host     = request_lines.last[1].split(':')[0]
+    @host     = request_lines[2][1].split(':')[0]
     @origin   = host
-    @port     = request_lines.last[1].split(':')[1]
+    @port     = request_lines[2][1].split(':')[1]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.